dev/com.ibm.ws.security.saml.sso_fat.common/src/com/ibm/ws/security/saml20/fat/commonTest/SAMLCommonTest.java

```diff
@@ -200,11 +205,30 @@ public static SAMLTestServer commonSetUp(String requestedServer,
 
     }
 
+    //issue 17687 
     public static SAMLTestServer commonSetUp(String requestedServer,
                                              String serverXML, String testType, String serverType,
                                              List<String> addtlApps, List<String> addtlMessages, Boolean checkForSecuityStart, String callbackHandler,
                                              String feature) throws Exception {
 
+        Map<String, String> cbHandlers = null;
+        if (callbackHandler != null && feature != null) {
+            cbHandlers = new HashMap<String, String>();
+            cbHandlers.put(callbackHandler, feature);
+            cbHandlers.put(callbackHandlerWss4j, featureWss4j);
+        }
```

Comment:
Thanks for pointing it out.  But is it possible to make the suggested change at a later time and with a github issue tracked for the time being?  I'm hoping to get this integrated to allow the Jakarta work to start.  So far only wssec saml FAT is calling this interface and does not have an impact to other FAT.  Hope to delay the update if I can.

Reply:
I've opened [https://github.com/OpenLiberty/open-liberty/issues/17717](url) to make the update and have assigned it to you Mei-hsaing.
I'll approve with the assumption that this will be updated.

Reply:
Thanks.